### PR TITLE
Update to Rollup v1

### PR DESCRIPTION
**Pull request by abdonrd**
Update to Rollup v1

### CODE REVIEW

File: rollup.config.js [hunk 1]
```diff
@@ -24,7 +24,6 @@ export default {
   plugins: [
     terser({
       warnings: true,
-      module: true,
```
Comment (jridgewell) [inline rollup.config.js]: Why this change?

Comment (abdonrd) [reply rollup.config.js]: Because it's the default now: [rollup-plugin-terser v4.0.0](https://github.com/TrySound/rollup-plugin-terser/releases/tag/v4.0.0)

>automatically set module: true for the esm format ([Tiger Oakes](https://github.com/NotWoods) in [#22](https://github.com/TrySound/rollup-plugin-terser/pull/22))
